src/net.cpp

```diff
@@ -2493,6 +2537,15 @@ void CConnman::StopNodes()
     if (fAddressesInitialized) {
         DumpAddresses();
         fAddressesInitialized = false;
+
+        if (m_use_addrman_outgoing) {
+            // Anchor connections are only dumped during clean shutdown.
+            auto anchors_to_dump = GetCurrentBlockRelayOnlyConns();
```

Comment:
I think explicitly writing the type here (`std::vector<CAddress>`) is clearer here.

Reply:
[Updated](https://github.com/bitcoin/bitcoin/pull/17428#issuecomment-706081630).